packages/block-editor/src/components/block-edit/context.js

```diff
@@ -1,29 +1,46 @@
-/**
- * External dependencies
- */
-import { noop } from 'lodash';
-
 /**
  * WordPress dependencies
  */
+import { useSelect } from '@wordpress/data';
 import { createContext, useContext } from '@wordpress/element';
 
-const Context = createContext( {
-	name: '',
-	isSelected: false,
-	focusedElement: null,
-	setFocusedElement: noop,
```

Comment:
`focusedElement` and `setFocusedElement` is context that has been removed a long time ago.